packages/document/main-doc/docs/en/guides/advanced-features/rspack-start.mdx

```diff
@@ -123,19 +123,49 @@ When Rspack is enabled for building through dev / build, the current version of
 
 You can override Rspack to a specific version using the capbilities provided by package managers such as pnpm, yarn, npm.
 
-For example, if you are using pnpm, you can update the Rspack version with `overrides` as shown below:
+For example, if you are using pnpm, you can override the Rspack version with `overrides` as shown below:
 
 ```json title=package.json
 {
   "pnpm": {
     "overrides": {
-      "@rspack/core": "nightly",
-      "@rspack/plugin-react-refresh": "nightly",
+      "@rspack/core": "latest",
+      "@rspack/plugin-react-refresh": "latest",
     }
   }
 }
 ```
 
+If you need to use the nightly/canary version of Rspack, the package name of the nightly/canary version of Rspack will be released after adding the `-canary` suffix, and all Rspack internal packages need to be overrided:
+
+```json title=package.json
+{
+  "pnpm": {
+    "overrides": {
+      "@rspack/binding": "npm:@rspack/binding-canary@nightly",
+      "@rspack/core": "npm:@rspack/core-canary@nightly",
+      "@rspack/cli": "npm:@rspack/cli-canary@nightly",
+      "@rspack/dev-server": "npm:@rspack/dev-server-canary@nightly",
+      "@rspack/plugin-minify": "npm:@rspack/plugin-minify-canary@nightly",
+      "@rspack/plugin-react-refresh": "npm:@rspack/plugin-react-refresh-canary@nightly",
+      "@rspack/plugin-preact-refresh": "npm:@rspack/plugin-preact-refresh-canary@nightly"
```

Comment:
Modern.js does not depend on `@rspack/plugin-preact-refresh`, `@rspack/cli`, `@rspack/dev-server` and `@rspack/plugin-minify`, can we remove them in our example

Reply:
done